packages/driver/src/cypress/stack_utils.ts

```diff
@@ -139,13 +139,21 @@ const captureUserInvocationStack = (ErrorConstructor, userInvocationStack?) => {
   if (!userInvocationStack) {
     const newErr = new ErrorConstructor('userInvocationStack')
 
+    userInvocationStack = newErr.stack
+
     // if browser natively supports Error.captureStackTrace, use it (chrome) (must be bound)
     // otherwise use our polyfill on top.Error
     const captureStackTrace = ErrorConstructor.captureStackTrace ? ErrorConstructor.captureStackTrace.bind(ErrorConstructor) : Error.captureStackTrace
 
     captureStackTrace(newErr, captureUserInvocationStack)
 
-    userInvocationStack = newErr.stack
+    // On Chrome 99+, captureStackTrace strips away the whole stack,
+    // leaving nothing beyond the error message. If we get back a single line
+    // (just the error message with no stack trace), then use the original value
+    // instead of the trimmed one.
+    if (newErr.stack.match('\n')) {
+      userInvocationStack = newErr.stack
```

Comment:
`captureStackTrace` modifies the error object in place. So for example, beforehand, `newErr.stack` might will be 15 lines. Then we call `captureStackTrace(newErr, captureUserInvocationStack)`, and now `newErr.stack` is only 10 lines long.

This guard is against the case - in Chrome 99 - where it was already only two lines, and gets reduced to one. Eg, it might be

```
Err: Foobar is bad!
  at (localhost:3500/cypress/stuff/myspec.js:54:12)
```

and it gets trimmed down to

```
Err: Foobar is bad!
```

I'm not 100% sure what caused the changed behavior of captureStackTrace in chromium 99, but the first stack trace is far more useful than the second.

Reply:
ahhh the fact `captureStackTrace` modifies this as well was the part I was missing! Thanks for the extra info.